config.ru

```diff
@@ -1,11 +1,24 @@
 require 'sinatra'
 require 'sinatra/reloader' if development?
+require 'open3'
+require 'erubi'
+
+set :erb, :escape_html => true
+
+helpers do
+  def parse_ps(ps_string)
+    #output_arr = ps_string.split(/\n/).map { |s| s.split(" ") }
+    ps_string.scan(/([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+(.*)/)
```

Comment:
Couple of other options for this huge regular expression to make it easier to swallow:

```ruby
ps_string.split("\n").map { |line| line.split(" ", 11) }
```

```ruby
ps_string.scan(/(\S+)\s+(\S+)\s+(\S+)\s+(\S+)\s+(\S+)\s+(\S+)\s+(\S+)\s+(\S+)\s+(\S+)\s+(\S+)\s+(.*)/)
```